Developers/Writing_Bears.rst

```diff
@@ -65,14 +65,15 @@ This bear is stored at ``./bears/HelloWorldBear.py``
 In order to let coala execute this bear you need to let coala know where
 to find it. We can do that with the ``-d`` (``--bear-dirs``) argument:
```

Comment:
the shortlog is quite unclear. try changing it to:

"Writing_Bears: Add ``--flush-cache`` to command"